### ISSUE TITLE
Build failed

### ISSUE THREAD

**Issue by Laxusgee** [jedld/lenddo-cordova#1]
I get this error 

BUILD FAILED
C:\android-sdk-windows\tools\ant\build.xml:577: ../google-play-services_lib reso
lve to a path with no project.properties file for project C:\repos\one-credit-mo
bile-app\platforms\android\com.lenddo.cordova.plugin\onecreditapp-LenddoSDK

anytime i do cordova build android

**Comment by jedld** [comment]
Thanks for the report, I will try to reproduce this issue and got back to you as soon as possible.

**Comment by jedld** [comment]
It seems to be an issue with the path on windows platforms.
For now please edit platforms/android/com.lenddo.cordova.plugin/LenddoSDK/project.properties

and change the slash:

android.library.reference.1=../google-play-services_lib

to

android.library.reference.1=..\google-play-services_lib

I'll figure out how to make it cross platform.

Also Please be informed that the plugin has been moved to https://lenddo.kilnhg.com/Code/public/android/lenddo-phonegap.git, please use that instead for future development

**Comment by Laxusgee** [comment]
thanks but https://lenddo.kilnhg.com/Code/public/android/lenddo-phonegap.git does not seem to exist

**Comment by jedld** [comment]
it uses kiln so it returns page not found when you try to access it directly fro the web. However you should still be able to git clone on it. 

As for the library reference, editing project.properties did not work? Please try using a full path instead (e.g. C:\repos\one-credit-mo
bile-app\platforms\android\com.lenddo.cordova.plugin\google-play-services_lib)

**Comment by Laxusgee** [comment]
i finally got it. instead of calling the i made a path to the folder ../folder-path

**Comment by jedld** [comment]
Thanks let me know if you are able to go through the whole Sign up flow. I've also pushed a few updates, make sure you update first if you have any issues.

**Comment by Laxusgee** [comment]
i was not able to sign in even after update...app keeps shutting after i click the lenddo button. this is my error log

http://pastebin.com/yHj2Cr9b

**Comment by jedld** [comment]
Thanks!

I was able to replicate your problem and noticed that cordova recently has quirks that needed workarounds (I did not encounter this before), I'll be adding those to the documentation. However to solve your problem, kindly do the following:

1. Navigate to your project folder. Add the android platform if you haven't done so already:

   cordova platform add android

2. Fetch the latest update for the plugin

   cordova plugin remove com.lenddo.cordova.plugin
   cordova plugin add https://lenddo.kilnhg.com/Code/public/android/lenddo-phonegap.git

3. Remove the custom_rules.xml file. (This is what causes your problem)
   
   For Mac/Linux

   rm platforms/android/custom_rules.xml 

  For Windows

  del platforms\android\custom_rules.xml

4. Build run your app normally (e.g.):

  cordova build android
  cordova run --debug
